esphome/components/st7789v/st7789v.h

```diff
@@ -126,6 +126,8 @@ class ST7789V : public PollingComponent,
 
   void write_display_data();
 
+  display::DisplayType get_display_type() override { return display::DisplayType::DISPLAY_TYPE_BINARY; }
```

Comment:
This is a coloured display...
```suggestion
  display::DisplayType get_display_type() override { return display::DisplayType::DISPLAY_TYPE_COLOR; }
```

Reply:
Oops. I thought I looked every display up to get the properties, but I messed this one up :-(